lib/package.py

```diff
@@ -56,6 +59,7 @@ def __init__(self, name):
         self.download_source = None
         self.install_dependencies = []
         self.build_dependencies = []
+        self.build_cache_dir = os.path.join(BUILD_CACHE_DIR, self.name)
```

Comment:
@gutoyr will work on changing the cache and results paths. Let's see if it's still needed after his changes.